src/Services/Identity/Identity.API/Controllers/ConsentController.cs

```diff
@@ -62,12 +62,12 @@ public async Task<IActionResult> Index(ConsentInputModel model)
             ConsentResponse response = null;
 
             // user clicked 'no' - send back the standard 'access_denied' response
-            if (model.Button == "no")
+            if (model != null && model.Button == "no")
```

Comment:
A shorter expression using language features:

```suggestion
            if (model?.Button == "no" ?? false)
```